index.html

```diff
@@ -47,9 +47,9 @@ <h5 class="card-title">project 1</h5>
                 <div class="card p-2 col-md-4 col-sm-12">
```

Comment:
It would be better if you wrap these lines with a main tag in order to make your code Semantic like you used the section. tag.